### ISSUE TITLE
Reproducing results with the pre-trained models

### ISSUE THREAD

**Issue by VenkateshParvathala** [AppleHolic/source_separation#29]
Hello,
   I hope that the results stated in the section 'Evaluation Scores' is evaluated on test set of voice bank corpus. It was told that the PESQ is around 2.3(With or Without audioset). But when I evaluated with the pretrained model named 'second_voice_bank.best.chkpt', I could only get a PESQ score around 1.80. I have few questions on this
1)May we know which evaluation data you were exactly using?
2)How to reproduce the results with pretrained model?
  Thanks

**Comment by AppleHolic** [comment]
1) I did normalizing volume for audio preprocessing. I think it could produce a different result.
2) If it did produce a different result, I think I missed data augmentation that fully downloaded audioset files. Unfortunately, I am not an employee of the company at that time, I cannot fully check the entire process at now.. I will decide to update or to archive this repository. Sorry to wait for a comment on this issue.

**Comment by AppleHolic** [comment]
If I check the entire training process sometimes, I will re-open this issue. Close it now.
